markdown.go

```
@@ -345,14 +347,35 @@ func (m *Markdown) Table(t TableSet) *Markdown {
+					},
+				},
+			),
+		),
+		tablewriter.WithConfig(tablewriter.Config{
+			Row: tw.CellConfig{
+				Alignment: tw.CellAlignment{Global: tw.AlignNone},
+			},
+		}),
+	)
+	table.Header(t.Header)
+	table.Bulk(t.Rows)
```

golangci-lint (1.64.8)

Error return value of `table.Bulk` is not checked

(errcheck)